src/main.rs

```diff
@@ -101,7 +103,12 @@ fn handle_end(end_req: Json<GameState>) -> Status {
 
 fn main() {
     let address = "0.0.0.0";
-    let port = 8080;
+    let env_port = env::var("PORT").ok();
+    let env_port = env_port
+        .as_ref()
+        .map(String::as_str)
+        .unwrap_or("8080");
+    let port = env_port.parse::<u16>().unwrap();
```

Comment:
this will error out if port is defined as a non-u16 but I think that'll be clear from stderr

Reply:
Cool - I know very little about Rust and was hoping I didn't commit too many rust-related crimes! Thanks for the review.